fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACFilter.java

```diff
@@ -191,7 +211,16 @@ private boolean isAuthorized(final Subject currentUser, final HttpServletRequest
                 }
             }
         case "POST":
-            if (currentUser.isPermitted(toWrite)) {
+            final String slug = httpRequest.getHeader("Slug");
+            if (isAcl || (slug != null && slug.endsWith(FCR_ACL))) {
```

Comment:
I'm not sure I understand what is going on here:
1. ACLs don't support `POST`, as they are not containers, so I don't believe the `isAcl` case needs to be considered
2 The fcrepo code makes a hard assumption that the path of acls ends with `FCR_ACL`.  Since this is so, any `Slug` that suggests creating a resource with such a path ought to be ignored, or rejected.